boards/opentitan/README.md

```diff
@@ -28,24 +28,24 @@ or simulator built from that version of the OpenTitan codebase.
 Programming
 -----------
 
-The latest supported version (commit SHA) of OpenTitan is specified in the [OPENTITAN_SUPPORTED_SHA](https://github.com/tock/tock/blob/master/boards/opentitan/earlgrey-cw310/Makefile) make variable found:
->  boards/opentitan/earlgrey-cw310/Makefile
-
-In *general* it is recommended that users start with the commit specified by `OPENTITAN_SUPPORTED_SHA` as newer
+It is recommended that users start with the commit specified by `Earlgrey-M2.5.2-RC0` as newer
 versions **have not been** tested.
```

Comment:
We're still using a SHA from https://github.com/tock/tock/pull/3639

Reply:
Who is we? This is what the current Makefile has:

```shell
# OpenTitan commit 21ce4e9761abdf5c919b46e5ae64a5a8e24992f7 is Earlgrey-M2.5.2-RC0.
# This commit is what is being taped out in the first Earlgrey chip.
OPENTITAN_SUPPORTED_SHA := 21ce4e9761abdf5c919b46e5ae64a5a8e24992f7
```

Reply:
Tock, we are using the taped out version but we will still need to specify a SHA hash.

Reply:
Yeah, we are using the tag from `Earlgrey-M2.5.2-RC0`. The specific hash is specified in the Makefile

Reply:
No I'm saying we can't use that tag we need to use this commit: https://github.com/tock/tock/pull/3639#issuecomment-1724140257

Reply:
I'm not changing the tag here. It's the same one we are already using. If there is a new one to use we can update to that

Reply:
Awe! Is your point that we shouldn't use a release tag and instead should continue to use SHAs?

Reply:
I think the relevant discussion is in that PR, but the gist is that timing for FPGA can still change without changing the taped out HW.

Reply:
I feel like that should be discussed in that PR. Currently we are using a release tag, so I think it makes sense to say that in the README. If we decided to change we can just update the README

Reply:
I don't want to merge this PR until after that one, so we can just wait. I was just hinting to preemptively remove the conflict.

Reply:
Why wait on https://github.com/tock/tock/pull/3639? It doesn't even apply and this PR is ready and fixes broken test cases

Reply:
Backout the readme changes so this just updates the tests? That makes this an easy merge.

Reply:
Done!